front/components/providers/AI21Setup.tsx

```diff
@@ -67,7 +80,7 @@ export default function AI21Setup({ owner, open, setOpen, config, enabled }) {
 
   return (
     <Transition.Root show={open} as={Fragment}>
-      <Dialog as="div" className="relative z-10" onClose={() => setOpen(false)}>
+      <Dialog as="div" className="relative z-30" onClose={() => setOpen(false)}>
```

Comment:
FMI, why do we need to move to `z-30` everywhere ?

Reply:
Standardizing all Dialogs as some of them woudl conflict with the AppLayout title :+1: